### ISSUE TITLE
Frozen Screen and uncontrollable after direct

### ISSUE THREAD

**Issue by Waschmaschinenfreund** [flybywiresim/aircraft#9201]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2024-10-31T14:32:39+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "f26029363ad3edeeafbed20aa79c13b32c8090f8",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-HEAD.f260293"
}
```

### Describe the bug

After Inserting a Direct To, the PFD and ND froze, only the standy instrument was working. I was not able to control the aircraft anymore (neither AP nor hand flying). The plane was turning and climbing/descending randomly, loosing up to 10000 Feet altitude. Also the Direct To waypoint was not showing, only the turn to next waypoint thereafter

### Expected behavior

Plane should fly direct as normal, and stay controllable

### Steps to reproduce

Do a direct. I was about one hour into the flight

(Direct was done via DIR TO on FPL page)

### References (optional)

![Screenshot 2024-10-31 185508](https://github.com/user-attachments/assets/f0c892e6-e98c-413b-8525-115d50d526ea)
![Screenshot 2024-10-31 190652](https://github.com/user-attachments/assets/98c43818-89dc-4616-8a97-c6267a0b6a96)
![Screenshot 2024-10-31 190706](https://github.com/user-attachments/assets/e9d07e44-7d19-421d-b7ee-803d91032c58)

### Additional info (optional)

First flight with the A380.

A friend of mine had the same problem.

https://fly.volanta.app/flights/d8a559cb-6b52-4bc5-ac02-328b0ab72cf5

### Discord Username (optional)

Marcel Davis 1&1

**Comment by Waschmaschinenfreund** [comment]
Did another flight. This time I did not do a direct, but after some time the same problem happend again...

**Comment by Toasty3920** [comment]
Same issue on my side. Also did a direct and after that the PFD and ND froze too together with the flight controls (Manual flying and AP)

**Comment by aubinschneider13** [comment]
Same for me, and in addition I had some doors that kept opening and closing randomly. I saw on discord that a lot of people had this issue
![image](https://github.com/user-attachments/assets/e9053397-f06e-47bb-99c9-fce3b9b58adf)

**Comment by Toasty3920** [comment]
My rear doors have been opened aswell, but that was due to GSX opening them on ground, so they couldn't be closed anymore then.

**Comment by Waschmaschinenfreund** [comment]
Yeah I believe it has to be linked to the doors, I had that issue just before doing the direct

**Comment by Waschmaschinenfreund** [comment]
I was able to reproduce the door issue, by going to performance tab in the efb, and then back to the service menu. after repeating once more the doors opened. You can close them back by swiching to Fuel and back to Service

**Comment by N1K340** [comment]
Same thing, everything froze including flight controls.
Completely unusable after that
